src/plugins/ga.js

```diff
@@ -1,31 +1,61 @@
 /* eslint-disable no-console */
 // From https://github.com/egoist/vue-ga/blob/master/src/index.js
-function appendScript() {
+
+function appendScript(id) {
   const script = document.createElement('script');
   script.async = true;
-  script.src = 'https://www.google-analytics.com/analytics.js';
+  script.src = 'https://www.googletagmanager.com/gtag/js?id=' + id;
   document.body.appendChild(script);
 }
 
-function init(id) {
-  appendScript();
-  window.ga =
-    window.ga ||
+// global site tag instance initialized
+function initGlobalSiteTag(id) {
+  appendScript(id);
+
+  window.dataLayer = window.dataLayer || [];
+  window.gtag =
+    window.gtag ||
     function () {
-      (window.ga.q = window.ga.q || []).push(arguments);
+      window.dataLayer.push(arguments);
     };
 
-  window.ga.l = Number(new Date());
-  window.ga('create', id, 'auto');
+  window.gtag('js', new Date());
+  window.gtag('config', id);
+}
+
+// add additional products to your tag
+// https://developers.google.com/tag-platform/gtagjs/install
+function initAdditionalTag(id) {
+  window.gtag('config', id);
+}
+
+function init(ids) {
+  if (Array.isArray(ids)) {
+    // set the first id to be a global site tag
+    initGlobalSiteTag(ids[0]);
+
+    // the rest ids
+    ids.forEach((id, index) => {
+      if (index > 0) {
+        initAdditionalTag(id);
+      }
+    });
+  } else {
+    initGlobalSiteTag(ids);
+  }
 }
 
 function collect() {
   if (!window.ga) {
     init($docsify.ga);
   }
 
-  window.ga('set', 'page', location.hash);
-  window.ga('send', 'pageview');
+  // usage: https://developers.google.com/analytics/devguides/collection/gtagjs/pages
+  window.gtag('event', 'page_view', {
+    page_title: document.title,
```

Comment:
Hi, there is some questions:
1. Do we still need to keep `ga.js` ?
2. `docs/plugins.md` is necessary to keep the instructions for `ga` and add the instructions for `gtag` ?

Reply:
Great~

Reply:
I just pushed the code, there is an exception in ci/codesandbox. I'm not sure what the reason is, please check it for me.

Reply:
I took a look and tried a few things in #1851 but I was unable to find the issue. This happened once before with Codesandbox it is it kinda of a PITA to troublehoot because the failures don't happen locally or in CI tests.

As much as I'd love to wrap this up, I don't have the spare cycles to devote to it right now. I will circle back as soon as I can devote some time to it.

In the meantime, thanks for being patient with this, @ekoz. Very much appreciated. :smile: